### PR TITLE
Adds Cryostatis Shotgun Darts

### PR DESCRIPTION
**Pull request by MMMiracles**
:cl: MMMiracles
add: You can now produce a cryostatis variant of the shotgun dart after researching Medical Weaponry. Holds 10u and doesn't have reagents react inside it.
/:cl:

[why]: Shooting people with tiny water+potassium bombs or whatever other insidious things that could be done with 10u of space.

I'm not sure if this should be something medical or security gets, since security would probably have better use for shotgun shells but medical actually has chemical access and its based on medical research.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
Doesn't sound super OP and still fun enough

**Comment by PKPenguin321** [comment]
Gibbing bomb darts yes

**Comment by MMMiracles** [comment]
i severely underestimated the power of water+potassium bombs

7/7 water/potassium bombs are an insta-delimb on regular crew and drops security in 1, kills in 4 with all their shit being in shreds

oops

**Comment by praisenarsie** [comment]
@MMMiracles ied attached to spears are insta delimb on crew and security

🥘

**Comment by Bawhoppen** [comment]
Precedent doesn't really matter on balance. Revolvers do like 40 brute a shot, yet we don't see new guns being added like that.

**Comment by praisenarsie** [comment]
wait a second, we don't we add a screen shake when you fire a revolver

**Comment by Bawhoppen** [comment]
Revolvers were unrelated example

**Comment by Iamgoofball** [comment]
You're retarded bawhoppenn

**Comment by PKPenguin321** [comment]
old shotgun darts did this and could instagib with black powder iirc
baw if you think reliably mass producible instagib guns is the same as a 13 TC traitor only item youre dumb

**Comment by MMMiracles** [comment]
Shotgun darts hold 30u, which is considerately more. I did some local testing with some explosive mixtures and 10u is small enough to where if you want to be doing stuff like blowing off limbs you need to be hitting the target at least twice.

**Comment by Bawhoppen** [comment]
Okay can someone explain the issue with what I said? The first statement or the second?
If the first, then I don't know about balance on this particular issue, so maybe I was just wrong.
If the second, then my point still stands about not using OP existing stuff to justify adding new OP stuff.

**Comment by PKPenguin321** [comment]
i guess the first thing you said was right but the revolver comparison was silly

**Comment by Bawhoppen** [comment]
Sorry, I guess I could have explained better.

**Comment by ExcessiveUseOfCobblestone** [comment]
I was hoping it would use @Iamgoofball 's freezing effect for Freon but this works too lol

File: code/modules/projectiles/ammunition/ammo_casings.dm [hunk 1]
```diff
@@ -283,12 +283,24 @@
...
+	reagent_amount = 10
+	reagent_react = FALSE
 
 /obj/item/ammo_casing/shotgun/dart/Initialize()
 	. = ..()
 	container_type |= OPENCONTAINER
-	create_reagents(30)
-	reagents.set_reacting(TRUE)
+	create_reagents(reagent_amount)
+	if(reagent_react)
```
Comment (vuonojenmustaturska) [inline code/modules/projectiles/ammunition/ammo_casings.dm]: You are guaranteed to always have the var at least. `reagents.set_reacting(reagent_react)` please, skip the if.

Comment (optimumtact) [reply code/modules/projectiles/ammunition/ammo_casings.dm]: do the needful

**Review by WJohn — DISMISSED** [review]

**Pull request merged**